### PR TITLE
Added 5 new patterns

### PR DESCRIPTION
**Pull request by notsolucky**
in order Honeycomb, Wave, Tree, Pyramid and Chocolate Weave
Preview on dabblet.com: http://dabblet.com/gist/6924eace082562570caf

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -338,7 +338,62 @@
...
+linear-gradient(135deg, transparent 91%, #3c5f49 91%), 
+linear-gradient(225deg, transparent 88%, #3c5f49 88%), 
+linear-gradient(135deg, transparent 88%, #3c5f49 88%), 
+linear-gradient(225deg, transparent 87%, #3c5f49 87%), 
+linear-gradient(135deg, transparent 87%, #3c5f49 87%), 
+linear-gradient(262deg, transparent 92%, #4b3b2a 92%), 
+linear-gradient(99deg, transparent 92%, #4b3b2a 92%);
+background-size: 60px 100px;
+background-position: -100px -40px, -100px -40px, -100px -30px, -100px -30px, -100px -20px, -100px -20px, -100px -10px, -100px -10px, -102px 0, -98px 0;"
+title="Tree" data-author="Paul Salentiny"></li>
```
Comment (LeaVerou) [inline index.html]: This one is way too big and not useful enough to be worth it.

File: index.html [hunk 2]
```diff
@@ -338,7 +338,62 @@
...
+
+<li style="background-color: #000;
+background-image: linear-gradient(315deg, transparent 75%, #d45d55 75%), 
+linear-gradient(45deg, transparent 75%, #d45d55 75%), 
+linear-gradient(135deg, #a7332b 50%, transparent 50%), 
+linear-gradient(45deg, #6a201b 50%, transparent 50%), 
+linear-gradient(45deg, transparent 50%, #561a16 50%);
+background-size: 20px 20px;
+background-position: -10px 0, -10px 0, 0 0, 0 0, 0 0;"
+title="Pyramid" data-author="Paul Salentiny"></li>
```
Comment (LeaVerou) [inline index.html]: I wonder if we could set a base color and specify the rest as semi-transparent black.
Also, I’m pretty sure the number of gradients can be reduced, but don’t have enough time to do it myself right now.

File: index.html [hunk 3]
```diff
@@ -338,7 +338,62 @@
...
+
+<li style="background: 
+linear-gradient(45deg, #cdbb99 12.5%, transparent 12.5%, transparent 88.5%, #cdbb99 88.5%), 
+linear-gradient(135deg, transparent 37.5%, #a37e58 37.5%, #a37e58 63.5%, transparent 63.5%), 
+linear-gradient(45deg, transparent 37.5%, #cdbb99 37.5%, #cdbb99 63.5%, transparent 63.5%);
+background-color: #755C3B;background-size: 25px 25px;"
+title="Chocolate Weave" data-author="Paul Salentiny"></li>
+
```
Comment (LeaVerou) [inline index.html]: This is very nice!

**Comment by LeaVerou** [comment]
Hey, sorry for the delay. I left a few comments on them.

Also, could you please set the color stop positions of every stop that has the same position as the previous one to `0`? this way they can be changed in one place only, and the browsers adjusts them to be the same. E.g. `linear-gradient(red 50%, white 50%)` would become `linear-gradient(red 50%, white 0)`.
Thanks!

**Comment by notsolucky** [comment]
No Problem, I am busy right now as well.
As you suggested, I quickly changed the color stops and removed one superfluous gradient on the pyramid pattern among other little changes.

**Comment by LeaVerou** [comment]
Sorry for the 1 year delay, merged it now, thanks!

**Pull request merged**